ext/spl/php_spl.stub.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+function iterator_to_array(Traversable $iterator, bool $use_keys = true): array {}
+
+function iterator_count(Traversable $iterator): int {}
+
+function iterator_apply(Traversable $iterator, callable $function, ?array $args = null): ?int {}
+
+function class_parents(object|string $instance, bool $autoload = true): array|false {}
+
+function class_implements(object|string $what, bool $autoload = true): array|false {}
+
+function class_uses(object|string $what, bool $autoload = true): array|false {}
```

Comment:
`object|string` union is not yet supported based on this: https://github.com/php/php-src/commit/292a1aeb59f348aabed58ac4fbc1d467dde3fcea (see the description as well)